.travis.yml

```diff
@@ -1,39 +1,29 @@
 language: julia
-
 os:
-  - osx
   - linux
-
+  - osx
+  - windows
 julia:
-  - 1.0
```

Comment:
Why remove?

Reply:
Errors the `test/Project.toml` / `docs/Project.toml`. Since the example is avoiding the experimental and will do the LTS flavor I can add it back.